### DISCUSSION TITLE
Fancontrol's reach to all areas of the PC

### DISCUSSION THREAD

**Discussion by hokkaidojef** [Rem0o/FanControl.Releases#2263] [Q&A]
Hi Everyone!

I know little about coding, but I am looking for a way to keep my fans controlled in windows 11 on a nvme and 
still carry over the same function on my sata ssd (using Linux Mint) on the same PC. Something in Task Scheduler
or such, to universalized fancontrols reach into my secondary drive on the PC?
Right now Windows side slows to proper speeds and Linux on the other drive is at 100% speed.

Basically can fancontrol control the fan speeds
for the whole PC regardless of what drive and OS ?
My thinking is that both OSes and Drives are sharing the same CPU/GPU AMD Ryzen 5G
and MSI Tomahawk B550 motherboard.
Please forgive any lack of knowledge, hours of Google didn't help.

Any insight is well appreciated.
Jeff :)

**Comment by Rem0o** [comment]
FanControl is a software running on Windows. Hence Windows must be actively running for FanControl to work its thing.  You can't have it work running Linux.
